test/plotrecipes.jl

```diff
@@ -304,4 +315,12 @@ using ColorTypes
     fig, ax, _ = M.volumeslices(A3abc; x=:c)
     fig, ax, _ = M.volumeslices(A3abc; z=:a)
     M.volumeslices!(ax, A3abc;z=:a)
+
+    A3intervals1 = rand(X(10:1:15; sampling=Intervals(Start())), Y(1:3), Dim{:C}(10:15))
+    M.plot(A3intervals1; z=:C)
+    # Broken from here
+    A3intervals2 = rand(X(10:1:15; sampling=Intervals(Start())), Y(1:3), Z(10:15; sampling=Intervals(Start())))
+    M.plot(A3intervals2)
```

Comment:
oh... it's also slices of heatmaps? that would be more cumbersome. It will require a smart way of combining 

`heatmap(x,y,z; transformation = (:xy, value)`
`heatmap(x,y,z; transformation = (:xz,value)`
`heatmap(x,y,z; transformation = (:yz, :value)`

ok...please clarify the expected output.

Reply:
Yeah the idea is two intervals combine to make a square.

So like slices of heat maps, or just all the polygons that would make the heatmap. Not sure how to do it best